packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -786,47 +733,52 @@ export async function startBrowser(
   runCounter++;
   const { webdriverOptions, wdioOptions } = await processCommonOpts();
 
-  const browser: CompassBrowser = (await remote({
+  const options: RemoteOptions = {
     capabilities: {
-      browserName: BROWSER_NAME, // 'chrome' or 'firefox'
-      // https://webdriver.io/docs/driverbinaries/
-      // If you leave out browserVersion it will try and find the browser binary
-      // on your system. If you specify it it will download that version. The
-      // main limitation then is that 'latest' is the only 'semantic' version
-      // that is supported for Firefox.
-      // https://github.com/puppeteer/puppeteer/blob/ab5d4ac60200d1cea5bcd4910f9ccb323128e79a/packages/browsers/src/browser-data/browser-data.ts#L66
-      // Alternatively we can download it ourselves and specify the path to the
-      // binary or we can even start and stop chromedriver/geckodriver manually.
-      // NOTE: The version of chromedriver or geckodriver in play might also be
-      // relevant.
-      browserVersion: 'latest',
+      browserName: COMPASS_WEB_BROWSER_NAME,
+      ...(COMPASS_WEB_BROWSER_VERSION && {
+        browserVersion: COMPASS_WEB_BROWSER_VERSION,
+      }),
     },
     ...webdriverOptions,
     ...wdioOptions,
-  })) as CompassBrowser;
+  };
 
-  if (TEST_ATLAS_CLOUD_EXTERNAL_URL) {
-    // Navigate to a 404 page to set cookies
-    await browser.navigateTo(`https://${TEST_ATLAS_CLOUD_EXTERNAL_URL}/404`);
+  debug('Starting browser via webdriverio with the following configuration:');
+  debug(JSON.stringify(options, null, 2));
+
+  const browser: CompassBrowser = (await remote(options)) as CompassBrowser;
+
+  if (TEST_ATLAS_CLOUD_EXTERNAL) {
+    // To be able to use `setCookies` method, we need to first open any page on
+    // the same domain as the cookies we are going to set
+    // https://webdriver.io/docs/api/browser/setCookies/
+    await browser.navigateTo(`${TEST_ATLAS_CLOUD_EXTERNAL_URL!}/404`);
```

Comment:
@syn-zhu FYI, dropping the protocol from the url as this is a _URL, not just a domain